sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/v2/model/dto/SimpleResponseWrapper.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.docsapi.api.v2.model.dto;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+/**
+ * Response wrapper used for those Document API endpoints that do not need information provided by
+ * document response wrapper..
```

Comment:
A bit confusing given that `DocumentResponseWrapper` does not exist yet... Is there a way we can remember to replace this with a link later?

Reply:
Should I introduce the `DocumentResponseWrapper` right away here in the code?

Reply:
Why not, it can stay empty until we migrate features that use it.

Reply:
Done.